packages/plugin/src/schema.ts

```diff
@@ -30,27 +30,25 @@ export function getSchema(options: ParserOptions, gqlConfig: GraphQLConfig): Gra
 
   // We first try to use graphql-config for loading the schema, based on the type of the file,
   // We are using the directory of the file as the key for the schema caching, to avoid reloading of the schema.
-  if (options && options.filePath && !options.skipGraphQLConfig) {
+  if (gqlConfig && options?.filePath) {
     const fileDir = dirname(options.filePath);
 
     if (schemaCache.has(fileDir)) {
       schema = schemaCache.get(fileDir);
     } else {
-      if (gqlConfig) {
```

Comment:
Here, we can check for this earlier, we can't have something in the cache if we don't even have a graphql config.

Reply:
Perfectly right, thanks!